### PR TITLE
mark php and java dogstatsd clients as supported

### PR DESCRIPTION
**Pull request by irabinovitch**

### CODE REVIEW

File: content/libraries.md [hunk 1]
```diff
@@ -6,8 +6,12 @@ sidebar:
...
       href: "#python"
+    - text: PHP
+      href: "#php"
     - text: Ruby
       href: "#ruby"
+    - text; JAVA
+      href: "#java"
     - text: Go
       href: "#go"
     - text: C#
```
Comment (irabinovitch) [inline content/libraries.md]: should probably alphabetize these.

Comment (jyee) [inline content/libraries.md]: Agreed. Merge this first, then do another PR to alphabetize or do you want to alphabetize in this PR? This is good to merge if you wanted to do the former (merge, then new PR to alphabetize)

Comment (irabinovitch) [inline content/libraries.md]: ya will do a in a future PR.
